### PR TITLE
Add simple-omero-client fiji plugin

### PR DESCRIPTION
**Pull request by lldelisle**
Add simple-omero-client fiji plugin

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/fiji-simple_omero_client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+test:
+  files:
+    - test.groovy
+
+about:
+  home: https://github.com/GReD-Clermont/simple-omero-client
+  license: GPLv2
+  license_file: LICENSE.txt
+  summary: Fiji/ImageJ plugin to simplify API to put/get objects on an OMERO server.
```
Comment (ppouchin) [inline recipes/fiji-simple_omero_client/meta.yaml:29]: It is actually not specific to ImageJ/Fiji (although this package is), and does not provide any plugin in fact. 
Would it make sense to rephrase it as such?
```yaml
summary: Java library to simplify API to put/get objects on an OMERO server from Fiji/ImageJ.
```

**Review by ppouchin — CHANGES_REQUESTED** [review]

File: recipes/fiji-simple_omero_client/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+outdir=$PREFIX/share/plugins/
```
[the remark this answers is not in the document]
Comment (ppouchin) [reply recipes/fiji-simple_omero_client/build.sh]: I'm under the impression that libraries should go in the `jars` folder, but I'm not 100% sure.

[the remark this answers is not in the document]
Comment (lldelisle) [reply recipes/fiji-simple_omero_client/build.sh]: I think it is fine to do it like this.

**Comment by ppouchin** [comment]
Sounds good to me, but I guess the build will only pass once the dependency is uploaded to the repo.

**Comment by lldelisle** [comment]
@BiocondaBot please add label

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Comment by lldelisle** [comment]
Thank you so much @bgruening

**Pull request merged**